### ISSUE TITLE
Add ability to make a list of items to be placed down

### ISSUE THREAD

**Issue by jiaaluo** [necessary129/minecraft-autotorch#2]
This would be useful for modded fabric items such as new torch types, etc.

**Comment by necessary129** [comment]
Yeah. I though about how to gracefully implement this. My first thought was to see if there were a Torch item type. Then I could just check if the item was a torch type and then just use that. Then any modded item that inherits the torch item would automatically work.

I didn't really look into that but I should.

Other option would be to let the user define a set of items to be placed as you suggested. That is also good as it completely hands over the control to the user. Pretty good but the user can put in nonsensical items too.

What would you suggest the best option be?

**Comment by jiaaluo** [comment]
Sorry for the late reply, I believe adding the ability to create a setlist of torch items would be the best, there are also some tools in fabric mods that when you right-click it creates a torch block most notably from Kibe, the light ring so adding support for items that generate blocks would also be nice.
https://github.com/lucaargolo/kibe


This is just all I have on the topic other than to offer other than that some Minecraft utility clients have some nice X-Ray drop-down lists for adding blocks if you were to need a reference.
